modules/authentication/src/routes/routes.ts

```diff
@@ -337,70 +102,15 @@ export class AuthenticationRoutes {
         new ConduitRouteReturnDefinition('VerifyServiceResponse', {
           serviceId: ConduitString.Required,
           accessToken: ConduitString.Required,
-          refreshToken: ConduitString.Required,
+          refreshToken: config.generateRefreshToken ? ConduitString.Required : ConduitString.Optional,
```

Comment:
Refresh tokens are never generated for `config.generateRefreshToken === false`, therefore shouldn't the ternary operator affect the entire fields object, entirely omitting `refreshToken` whenever that's the case?